baras/deployments.go

```diff
@@ -170,9 +172,194 @@ var _ = Describe("deployments", func() {
 		})
 
 		It("completes the deployment", func() {
-			deploymentGUID := CreateDeployment(appGUID)
+			deploymentGUID := CreateDeployment(appGUID, "rolling")
 			Expect(deploymentGUID).ToNot(BeEmpty())
 			WaitUntilDeploymentReachesStatus(deploymentGUID, "FINALIZED", "DEPLOYED")
 		})
 	})
+
+	Describe("Canary deployments", func() {
```

Comment:
thought(non-blocking): Should we have similar describe block for rolling deployments?

Reply:
Some of the existing tests are not specific for rolling, so we'd need to do some surgery to unpick the strategy specific tests. For now I think we can leave it and revisit it if we add a future strategy.